Makefile

```diff
@@ -1,19 +1,19 @@
+img := gcr.io/istio-testing/build-tools:2019-07-31
+pwd := $(shell pwd)
+uid := $(shell id -u)
+run = docker run -t -i --sig-proxy=true --rm -v /etc/passwd:/etc/passwd:ro -v $(pwd):/work -w /work $(img)
+make = $(run) make -f Makefile.container.mk
+
 gen:
-	@go generate ./...
+	@$(make) gen
```

Comment:
What do you think of making a wildcard rule the takes anything and runs it in the docker one? This may not be needed here but for Istio we have 100s of rules

Reply:
Yeah, it was on my todo list to figure out how to do that. Make can do everything, so long as you know the secret language...

Reply:
```
%:
	$(make) $@
```

Reply:
Excellent, thanks.

Reply:
if you throw enough symbols at make something will work eventually :slightly_smiling_face: